app/code/Magento/Cms/Model/GetUtilityPageIdentifiers.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Model;
+
+use Magento\Cms\Api\GetUtilityPageIdentifiersInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Utility Cms Pages
+ *
+ * @api
```

Comment:
No need to mark class as an `@api` only interface is and API. Class as implementation is module internal details.